src/format.js

```diff
@@ -0,0 +1,105 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const STRIKE = 10;
+  const STRIKE_SYMBOL = 'X';
+  const SPARE_SYMBOL = '/';
+  let isParty = true;
+  let game = new Game(Frame);
+
+  let setTotal = () => {
+    document.getElementById('currentScore').innerHTML = game.score();
+  };
+
+  let individualScores = () => {
+    let collumnID = 1;
+    let scores = game.runningScores();
+    scores.forEach((score) => {
+      document.getElementById(`scoreFrame${collumnID}`).innerHTML = `${score}`;
```

Comment:
String interpolation - woo! That's the first time I've seen this used (it's new in ES6 and I haven't done a lot of coding recently). 💯 

More seriously, it's great that you're building the view dynamically and in DRY fashion. For next level javascript, think about how you could treat each of the view elements as a class/object itself. E.g:

```
function FrameView(element) {
   this.element = element;
}

FrameView.prototype.setScore(score) {
  this.element.innerHTML = score;
}
```